matrix_client/client.py

```diff
@@ -132,16 +132,25 @@ def upload(self, content, content_type):
             response = self.api.media_upload(content, content_type)
             return response["content_uri"]
         except MatrixRequestError as e:
-            raise MatrixRequestError(code=e.code, content="Upload failed: %s" % e)
+            raise MatrixRequestError(code=e.code,
+                                     content="Upload failed: %s" % e)
         except KeyError:
```

Comment:
This is probably fine, but in general when capturing general exceptions like this its best to keep the scope as small as possible. For example, in this case if a programmer error happened  in `.media_upload(...)` that caused it to raise a `KeyError` the app would incorrectly assert that it was due to the `content_uri` not existing, which can leas to mass confusion.